File: data/llama3_8B_ins_results/readable_format_HPEC/fortran/file262.f90
subroutine print_array(ni, c)
implicit none

DATA_TYPE, dimension(ni, ni) :: c
integer :: ni
integer :: i, j
do i = 1, ni
do j = 1, ni
write(0, DATA_PRINTF_MODIFIER) c(j, i)
if (mod(((i - 1) * ni) + j - 1, 20) == 0) then
write(0, *)
end if
end do
end do
write(0, *)
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file262.cpp
#include <stdio.h>
#define DATA_TYPE int
#define DATA_PRINTF_MODIFIER "%d "
void print_array(int ni, DATA_TYPE c[100][100]) {
  for (int i = 0; i < ni; i++) {
    for (int j = 0; j < ni; j++) {
      printf(DATA_PRINTF_MODIFIER, c[j][i]);
      if ((i * ni + j) % 20 == 0) {
        printf("
");
      }
    }
  }
  printf("
");
}
